bintangbuddy/config.py

```diff
@@ -1,24 +1,19 @@
-import os
 # Will need to add to this if they ever have a new gym in the area
 GYM_IDS = {
     "campbell": "17047",
     "dublin": "17100",
     "milpitas": "17043",
     "san carlos": "17099",
-    "sunnyvale": "17098"
+    "sunnyvale": "17098",
 }
 
-# LOL I'm sure this is safe... right??
-BEARER = os.getenv('BEARER', None)
-assert BEARER, "Did you set the BEARER env variable? See https://github.com/teoh/bintang-buddy/blob/main/README.md#getting-the-bearer-token"
-
+AUTH_URL = "https://app.bukza.com/api/client/create/16509"
```

Comment:
Nice. I spent the whole winter holidays last year poring over Chrome's inspect tool and couldn't find this. Very glad you did.